third_party/move/move-model/src/ast.rs

```diff
@@ -1188,30 +1225,27 @@ impl ModuleName {
     /// Determine whether this is a script. The move-compiler infrastructure uses MAX_ADDR
     /// for pseudo modules created from scripts, so use this address to check.
     pub fn is_script(&self) -> bool {
-        static MAX_ADDR: Lazy<BigUint> = Lazy::new(|| {
-            BigUint::from_str_radix("ffffffffffffffffffffffffffffffff", 16).expect("valid hex")
-        });
-        self.0 == *MAX_ADDR
+        self.0 == Address::Numerical(AccountAddress::new([0xFF; AccountAddress::LENGTH]))
```

Comment:
Perhaps make MAX_ADDR  part of AccountAddress

Reply:
Also I thought address was 32 bytes, I count 32 f's in the previous version which is 16 bytes. Was that a bug?

Reply:
Yes, it was in fact a bug, someone reported this in the Move repo. It only effected docgen for scripts however, which I think no one uses.

Reply:
Thx, please at to description as fixed as well.

Reply:
Maybe close https://github.com/aptos-labs/aptos-core/issues/7731